src/libsync/propagateuploadv1.cpp

```diff
@@ -176,12 +176,6 @@ void PropagateUploadFileV1::slotPutFinished()
 
     slotJobDestroyed(job); // remove it from the _jobs list
 
-    qCDebug(lcPropagateUpload) << job->reply()->request().url() << "FINISHED WITH STATUS"
```

Comment:
It moved in src/libsync/networkjobs.cpp for all network jobs, instead of reporting it in the propagation job (so that the right category is used).